codex/market.nim

```diff
@@ -31,6 +31,7 @@ type
   ProofChallenge* = array[32, byte]
 
   # Marketplace events -- located here due to the Market abstraction
+  MarketplaceEvent* = Event
```

Comment:
Shouldn't then all the events bellow be `of MarketplaceEvent`?

Reply:
Good catch, yes, but it is an alias, so that's why the compiler didn't complain. The main reason for doing this was so that implementations using the abstraction don't also need to import `pkg/ethers`. Will fix.